bylaws.md

```diff
@@ -168,8 +159,13 @@ All members must provide and monitor an email address which will serve as the so
 Emails from the board must be sent from an @texaslan.org (http://texaslan.org/) email account.
 Any communications through social media must follow an official announcement in email form.
 
-**Article X. Amendments**
+**Article X. Board Orders**
+
+Processes that aren't required to be in the Bylaws shall be placed in the [Board Orders Document](./board_orders.md), to where it defines extra processes these bylaws must follow.
```

Comment:
I would redraft this clause entirely; it needs more detail. Here's a suggestion, subject to how the process actually works.

* Processes and directives which are not Bylaws but which are required by the sitting Board are referred to as Board Orders.
* Board Orders may be enacted by a 2/3rds vote of the Board.
* Board Orders must be re-enacted upon election of a new Board.
* Board Orders may be repealed by a 2/3rds vote of the Board.
* Board Orders shall be promptly posted in the [Board Orders Document](./board_orders.md) and communicated to the General Membership through the official communications channels.